src/command.c

```diff
@@ -1170,3 +1170,57 @@ cmd_replaygain(int argc, char **argv, struct mpd_connection *connection)
 
 	return 0;
 }
+
+int
+cmd_partitionlist(int argc, char **argv, struct mpd_connection *conn)
+{
+    if (!mpd_send_listpartitions(conn)) {
+		printErrorAndExit(conn);
+    }
+
+    struct mpd_partition *part;
+    while ((part = mpd_recv_partition(conn)) != NULL) {
+        printf("%s\n", mpd_partition_get_name(part));
+        mpd_partition_free(part);
+    }
+
+	my_finishCommand(conn);
+    return 0;
+}
+
+int cmd_partitionmake(int argc, char **argv, struct mpd_connection *conn)
+{
+    if (!mpd_command_list_begin(conn, false)) {
+		printErrorAndExit(conn);
+    }
+
+    for (int i = 0; i < argc; ++i) {
+        if (!mpd_send_newpartition(conn, argv[i])) {
+            printErrorAndExit(conn);
+        }
+    }
+
+    if (!mpd_command_list_end(conn)) {
+		printErrorAndExit(conn);
+    }
+    return 0;
```

Comment:
Here, the `mpd_response_finish()` comand is missing, therefore mpc never evaluates whether this has succeeded.

Reply:
Fixed.